lib/Doctrine/Common/Collections/AbstractLazyCollection.php

```diff
@@ -268,7 +268,7 @@ public function indexOf($element)
     /**
      * {@inheritDoc}
      */
-    public function slice(int $offset, ?int $length = null) : array
+    public function slice(int $offset, ?int $length = null) : Collection
```

Comment:
What is the impact of this BC break on the OSS ecosystem? Is `->slice()` called often?

Reply:
I am not sure. Trying to think of a way to search for that on github somehow.